fetch-in-react/src/app.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import UserList from './user-list';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      users: [],
+      isLoading: true
+    };
+  }
+
+  componentDidMount() {
+    fetch('https://jsonplaceholder.typicode.com/users')
+      .then(response => response.json())
+      .then(data =>
+        this.setState(state => ({ users: data }))
+      );
+    this.setState({ isLoading: false });
```

Comment:
We have the data on line 16, so you can set both state properties on line 17. The way it is now, `isLoading` gets set to `false` before the data has arrived.

Reply:
Thanks Tim. That makes sense. I thought that fetch would finish and then code on line 19 would run after. I tried logging it to the console and I guess it just happened that code ran in order the way I wanted it to. Fetch is asynchronous and I need to remind myself that.